src/lambda_function/s3_event_config/app.py

```diff
@@ -0,0 +1,67 @@
+"""
+This Lambda app responds to an S3 event notification and starts a Glue workflow.
+The Glue workflow name is set by the environment variable `PRIMARY_WORKFLOW_NAME`.
+Subsequently, the S3 objects which were contained in the event are written as a
+JSON string to the `messages` workflow run property.
+"""
+import os
+import json
+import logging
+import boto3
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+SUCCESS = "SUCCESS"
+FAILED = "FAILED"
+
+logger.info("Loading function")
+s3 = boto3.resource("s3")
+
+
+def lambda_handler(event, context):
+    logger.info(f"Received event: {json.dumps(event, indent=2)}")
+    try:
+        if event["RequestType"] == "Delete":
```

Comment:
Could you point me to the list of supported `RequestType`?

Reply:
This is defined by us. So we can change this to be whatever we'd like if that makes sense.

Reply:
I was planning to add them to the README